src/fs/driver/initfs/initfs_dvfs.c

```diff
@@ -246,5 +246,5 @@ static struct dumb_fs_driver initfs_dumb_driver = {
 	.mount_end = initfs_mount_end,
 };
```

Comment:
Yep, the rest of usages could be probably be fixed in future, but this particular piece of code has been edited by you in this PR.